class-wc-calypso-bridge.php

```diff
@@ -122,13 +128,14 @@ public function __construct() {
 
 		}, PHP_INT_MAX, 3 );
 
+		// Include calypso-bridge-setup this class as early as possible.
+		include_once dirname( __FILE__ ) . '/includes/class-wc-calypso-bridge-helper-functions.php';
```

Comment:
@manospsyx I moved the helper functions up here, so they are always accessible (as the one time operations are dependant on them)